cosmosdb/client_endpoint.bal

```diff
@@ -292,14 +302,14 @@ public client class DataPlaneClient {
         return finalStream;
     }
 
-    # Delete a stored procedure.
+    # Deletes a stored procedure.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container which contains the stored procedure
     # + storedProcedureId - ID of the stored procedure to delete
-    # + resourceDeleteOptions - The `ResourceDeleteOptions` which can be used to add additional capabilities to the
-    #                           request
-    # + return - If successful, returns `DeleteResponse`. Else returns `Error`.
+    # + resourceDeleteOptions - The `cosmos_db:ResourceDeleteOptions` which can be used to add additional capabilities 
```

Comment:
Update this to be clear on what `cosmos_db:ResourceDeleteOptions` is, and replace "which" with "that".